### ISSUE TITLE
NTR: Cholesterol sulfate high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8258]
New term request Cholesterol sulfate high in blood (martir_100319160039):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000565
- label: Elevated circulating cholesterol sulfate concentration
- parents: [Abnormal circulating cholesterol concentration (HP:0003107)](https://hpo.jax.org/app/browse/term/HP:0003107)
- pmid: PMID:12730293
- def: The concentration of cholesterol sulfate in the blood circulation is above the upper limit of normal.
- comment: holesterol sulfate is quantitatively the most important known sterol sulfate in human plasma. Cholesterol sulfate is a component of cell membranes where it has a stabilizing role, e.g., protecting erythrocytes from osmotic lysis and regulating sperm capacitation. It is present in platelet membranes where it supports platelet adhesion. Cholesterol sulfate can regulate the activity of serine proteases, e.g., those involved in blood clotting, fibrinolysis, and epidermal cell adhesion. As a result of its ability to regulate the activity of selective protein kinase C isoforms and modulate the specificity of phosphatidylinositol 3-kinase, cholesterol sulfate is involved in signal transduction.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8258
- synonyms:
